src/machine.rs

```diff
@@ -128,8 +128,12 @@ pub struct Tag {
 }
 
 impl Provenance for Tag {
+    // We use absolute addresses in the `offset` of a `Pointer<Tag>`.
     const OFFSET_IS_ADDR: bool = true;
 
+    // We cannot err on partial overwrites, it happens too often in practice (due to unions).
```

Comment:
You can use doc comments. They will show up on the impl inside the docs. I believe before or after the docs from the trait.

Reply:
I don't think docs are even generated for this code, and I don't think we consistently use doc comments inside Miri, but I guess this is as good a time as any to start. ;) Thanks!